src/charm.py

```diff
@@ -575,9 +590,21 @@ def _on_schema_upgrade_action(self, event: ActionEvent) -> None:
 
     def _on_ingress_ready(self, event: IngressPerAppReadyEvent) -> None:
         self._update_workload(event)
+        self.openfga_relation.update_server_info(
+            address=self._get_address(event.relation),
+            scheme="http",
+            port=str(OPENFGA_SERVER_HTTP_PORT),
+            dns_name=self._domain_name,
+        )
```

Comment:
the same call is made on `ingress-revoked`, we could put it in a separate method

Reply:
the ingress relation will change in a following PR, I would like to keep it this way for now as it will have to change again.